svtools/bin/bedpesort

```diff
@@ -6,12 +6,12 @@ if [ ! -z "$1" ]
 then
 	if [ ! -z "$2" ]
 	then
-		awk '$0~"^#" { print $0; next } { print $0 | "sort -k1,1V -k2,2n -k3,3n -k4,4V -k5,5n -k6,6n" }' $1 >$2
```

Comment:
Why are the sorts in bedpesort being changes to non-numeric on the end coordinate of the second breakpoint?